crates/bench/src/lib.rs

```diff
@@ -1,15 +1,24 @@
 use async_fn::AsyncFn1;
+use js_sys::Reflect;
 use log::error;
 use std::cmp;
 use thousands::Separable;
 pub use wasm_bench_macro::wasm_bench;
+use wasm_bindgen::JsValue;
+use web_sys::Performance;
+
+pub fn performance() -> Performance {
```

Comment:
Unrelated but I think maybe we should get rid of this crate ... it's not really the way we've decided to do benchmarking and I'm struggling to understand when we'd use it.